bundles/org.openhab.ui.basic/src/main/java/org/openhab/ui/basic/internal/render/AbstractWidgetRenderer.java

```diff
@@ -166,7 +167,11 @@ protected String preprocessIcon(String originalSnippet, @Nullable String icon, b
         try {
             switch (iconSource.toLowerCase()) {
                 case ICON_SOURCE_OH:
-                    iconSnippet = getSnippet(ignoreState ? "icon_oh_no_state" : "icon_oh");
+                    if (ICON_SET_OH_CLASSIC.equals(iconSet) && ICON_NAME_NONE.equals(iconName)) {
```

Comment:
Is there a special reason why we apply this logic only for the classic icon set?

Reply:
In the Android app `none` is treat as "no icon", but for `f7:none` a request is made: https://github.com/openhab/openhab-android/blob/9a2a12726e2eed8a7bb810e2b3afd6e1f9535cbf/mobile/src/main/java/org/openhab/habdroid/model/IconResource.kt#L158

Reason for this is simple: The handling of `none` has been implemented when only classic existed and hasn't been adapted later. 

If https://github.com/openhab/openhab-core/issues/4042 will be implemented and all icons without explicit icon source will be prefixed in the rest API, a user would need to define `oh:classic:none` to hide the icon. So I suggest to handle `none` as "no icon" for all icon sets, even when that disables all icons called `none` that may exist in other icon sets.

WDYT?

Reply:
We are talking only about OH icon source or also other icon sources (material, framework7, ...) ?
I would suggest to keep that only for OH icon source but for any icon set of that source.

Reply:
Yes, makes sense. We have full control about all icon sets in the openHAB icon source.

Reply:
@lolodomo If it should apply to any OH icon source, I assume you will do an update to this PR?

Reply:
Yes, a small change is then needed to handle none for OH icon source whatever the icon set.

Reply:
@kaikreuzer : this is now changed.